projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -98,6 +102,7 @@ export class WorkAllocationService {
     if (!this.isWAEnabled()) {
       return of(null);
     }
+    console.log(`completeTask: completing ${taskId}`);
```

Comment:
There are quite a few console logs. Do we really need them?

Reply:
I want to leave them in for now. I've actually removed quite a lot that were previously there